### DISCUSSION TITLE
Does Champsim model page structure caches?

### DISCUSSION THREAD

**Discussion by arunkp1986** [ChampSim/ChampSim#401] [General]
I am new to Champsim, and I would like to use it for TLB study. I saw that Champsim has page table walker, I would like to know whether page structure caches are modelled in the current version of Champsim?

**Comment by ngober** [comment]
Yes, there are PSCLs for table levels 2 through 5. They are configurable in the PTW through the keys `"pscl5_set"`, `"pscl5_way"`, and similar.

**Reply by arunkp1986** [reply]
Thanks a lot for your help. I got this doubt since the default stats do not report anything on hits/misses of PSCLs. Thanks once  again

**Comment by shz0116** [comment]
Hi, I just wondering whether the current processors implement the PSC caches ? and if so, what are the typical configurations ? Thanks.
